### PR TITLE
APEXMALHAR-2058 #resolve #comment moved Serde to serde package and ad…

### PR DESCRIPTION
**Pull request by ilooner**
…ded simple byte array Serde implementation.

### CODE REVIEW

**Comment by ilooner** [comment]
@chandnisingh please review

**Comment by otterc** [comment]
@ilooner please rebase

**Comment by ilooner** [comment]
@chandnisingh rebased and fixed license headers

File: library/src/main/java/org/apache/apex/malhar/lib/utils/serde/SerdeByteArrayToByteArray.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ */
+package org.apache.apex.malhar.lib.utils.serde;
+
+import org.apache.commons.lang3.mutable.MutableInt;
+
+/**
+ * This is a simple pass through {@link Serde}. When serialization is performed the input byte array is returned.
+ * Similarly when deserialization is performed the input byte array is returned.
+ */
+public class SerdeByteArrayToByteArray implements Serde<byte[], byte[]>
```
Comment (otterc) [inline library/src/main/java/org/apache/apex/malhar/lib/utils/serde/SerdeByteArrayToByteArray.java]: Can we shorten the name to PassThruByteArraySerde

**Comment by ilooner** [comment]
@chandnisingh Changed class name

**Pull request merged**